Libraries/LibWebView/Application.h

```diff
@@ -61,10 +61,10 @@ class Application {
 
 protected:
     template<DerivedFrom<Application> ApplicationType>
-    static NonnullOwnPtr<ApplicationType> create(Main::Arguments& arguments, URL::URL new_tab_page_url)
+    static NonnullOwnPtr<ApplicationType> create(Main::Arguments& arguments, URL::URL new_tab_page_url, WebView::IsHeadless is_headless = WebView::IsHeadless::No)
```

Comment:
Instead of adding this as an argument here, we should just set `web_content_options.is_headless` in headless-browser's `Application::create_platform_options`

Reply:
D'oh yeah. That's what that's there for! That simplifies things a bit, thanks!